src/extraction/fathom/index.js

```diff
@@ -32,8 +32,22 @@ const FEATURE_DEFAULTS = {
 const PRODUCT_FEATURES = {
   image: {
     ...FEATURE_DEFAULTS,
+    /**
+     * @return the URL of an image resource
+     */
     getValueFromElement(element) {
-      return element.src;
+      /**
+       * Given a CSS url() declaration 'url("http://foo")', return 'http://foo'.
+       */
+      function urlFromCssDeclaration(declaration) {
+        return declaration.substring(5, declaration.length - 2);
+      }
+      if (element.tagName === 'IMG') {
+        return element.src;
+      }
+      // The other thing the ruleset can return is an arbitrary element with
+      // a CSS background image.
+      return urlFromCssDeclaration(getComputedStyle(element)['background-image']);
```

Comment:
A bit of a nit, but I've always found that function declarations at the top break the flow of the function and make it harder to read. The function name helps the readability, but a variable can do just as well without interrupting the order:

```js
const backgroundImage = getComputedStyle(element)['background-image'];
return backgroundImage.substring(5, background.length - 2);
```

Reply:
I wanted this to be [ScreechinglyObviousCode](http://wiki.c2.com/?ScreechinglyObviousCode). With magic numbers like 5 and -2, it's not otherwise screechingly obvious that what we're trying to do is pull the param out of `url("…")`.

Reply:
An extra variable name, then?

```js
const backgroundImage = getComputedStyle(element)['background-image'];
const backgroundImageUrl = backgroundImage.substring(5, background.length - 2); // "url('<image_url>')"
return backgroundImageUrl;
```

(Or not. This is very much yak shaving at this point.)